fastats/maths/beta_pdf.py

```diff
@@ -0,0 +1,33 @@
+
+from numpy import power
+from fastats.maths import Gamma
+
+
+def beta_pdf(x, alpha, beta):
+	"""
```

Comment:
Hey, welcome! 

Looks like you're using tabs instead of spaces for the docstrings. I would recommend changing your editor/IDE setting - most python editors have a special setting to ensure tabs get auto-converted.